packages/dds/tree/package.json

```diff
@@ -45,6 +45,7 @@
 		"test:mocha:verbose": "cross-env FLUID_TEST_VERBOSE=1 npm run test:mocha",
 		"test:snapshots:regen": "npm run test:mocha -- --snapshot",
 		"test:stress": "cross-env FUZZ_TEST_COUNT=20 FUZZ_STRESS_RUN=true mocha --ignore \"dist/test/memory/**/*\" --recursive \"dist/test/**/*.spec.js\" -r @fluidframework/mocha-test-setup",
+		"ts2esm": "ts2esm ./tsconfig.json ./src/test/tsconfig.json",
```

Comment:
I am really looking forward to comments in package.json some day.
We leave out ./tsconfig.esnext.json because we assume the same file set.
...and including it would cause ts2esm to do double work (since it doesn't have smarts to handle duplicate coverage).